### ISSUE TITLE
[Package-Deps] Unable to get loaded package 'linter-remark'

### ISSUE THREAD

**Issue by dionlarson** [wooorm/linter-remark#9]
The Atom developer console shows:
```
[Package-Deps] Unable to get loaded package 'linter-remark'(anonymous function) @ ~/.atom/packages/linter-remark/node_modules/atom-package-deps/lib/helpers.js:59
```

Installed via `apm install linter-remark`, `linter-remark@4.0.0`, `atom@1.16.0`, `macOS Sierra 10.12.4`.

**Comment by wooorm** [comment]
@Arcanemagus Sorry to bug, but do you have any idea why that’s happening? I’ve got no clue!

**Comment by Arcanemagus** [comment]
@wooorm Anytime! 😉

It looks like you are missing the `package-deps` field in `package.json` defining the list of packages you need installed.

**Comment by wooorm** [comment]
Thanks @Arcanemagus! This should be fixed now, in 4.0.1, @dionlarson!
